src/modelbench/benchmarks.py

```diff
@@ -42,15 +42,18 @@ def __repr__(self):
 
 class BenchmarkDefinition(ABC, HasUid):
     _hazards: list[HazardDefinition]
-    _uid_definition = {"class": "self", "version": "0.5"}
 
-    def __init__(self, hazards: List[HazardDefinition]):
+    def __init__(self):
         super().__init__()
-        self._hazards = hazards
+        self._hazards = self._make_hazards()
 
     def hazards(self) -> List[HazardDefinition]:
```

Comment:
Is this a `property`?

Reply:
It's currently a read-only property, yes.